test/src/main/resources/log4j2-test.properties

```diff
@@ -143,6 +143,6 @@ logger.35.level = info
 logger.36.name = org.apache.thrift.transport.TIOStreamTransport
 logger.36.level = error
 
-rootLogger.level = debug
+rootLogger.level = trace
```

Comment:
I'm thinking we probably don't want to leave this change in, as that could easily fill up disk during a build on a resource constrained system, or create a lot of unnecessary IO wear on a developer's SSD. Debug level is probably fine for the ITs.

Reply:
Yes, that's true. It's definitely helpful to see what's happening in the test. I will definitely remove before a merge.